docs/blog/2022.md

```diff
@@ -0,0 +1,50 @@
+
+It's been an amazing year for envd! We've increased our incredible contributor base by nearly 60 people from all around the globe, and have gained a whopping 1,400 stars on GitHub. To top it off, 85 releases with three major updates including various new features and improvements have been released. We’ve also had in excess of 20,000 Docker Hub image downloads so far - giving us over 20k successful development environment builds this past 6 months alone.
+
+We couldn't have done it without our amazing community! Our [roadmap](https://envd.tensorchord.ai/community/roadmap.html) and [contributor guide](https://envd.tensorchord.ai/community/contributing.html) are constantly being updated to ensure that everyone is on the same page when it comes to building envd together. Thank you all for your support -- we truly appreciate it.
+
+At TensorChord, we strive to be open and transparent in all aspects of our work. To extend this ethos even further, we’ve made available a set of internal guidelines (https://internals.tensorchord.ai/) that outline how we help our new members onboard -  so people can benefit from understanding more about us! We welcome everyone interested in learning or contributing.
+
+## envd
+
+envd takes the hassle out of setting up a development environment for AI/ML.  Don't get bogged down in tedious configuration - let envd make it easy to focus on your work!
+
+Development environments are full of python and system dependencies, CUDA, BASH scripts, Dockerfiles, SSH configurations, Kubernetes YAMLs, and many other clunky things that are always breaking. envd is to solve the problem.
+
+With envd, your development process can be more efficient. It enables you to create tight-knit and easily reusable workflows with just a few lines of code. In no time at all, running 'envd up' will have you training models in an isolated environment - tapping into the power of modern development tools without any extra hassle.
+
+```python
+def build():
+base(os="ubuntu20.04", language="python3")
+# Configure the pip index if needed.
+# config.pip_index(url = "https://pypi.tuna.tsinghua.edu.cn/simple")
+install.python_packages(name = [
+"numpy",
+])
+shell("zsh")
+```
+
+Check out our official [documentation](https://envd.tensorchord.ai/) for more information or join us on [Discord](https://discord.gg/KqswhpVgdU) and let's talk about it together. See you there!
+
+## Future plans
+
+As we enter 2023, there is still plenty of work to be done in order for us to fully optimize the system. We are dedicating our focus on building performance, as well as fine-tuning and serving large models - all essential tasks that will help drive further progress this year!
```

Comment:
dedicating our focus on building performance -> dedicating our focus to build speed